machine/completion.md

```diff
@@ -4,6 +4,8 @@ keywords: machine, docker, orchestration, cli, reference
 title: Command-line completion
 ---
 
+{% assign machineversion = '0.12.2' %}
```

Comment:
Is there a way to define variables upstream? This variable is also defined in _machine/install-machine.md_

Reply:
We can use a variable in the files, that's as automated as we've gotten right now. I'll add that as a part of your PR.

Reply:
@albers we can define variables in the `config.yml` but they were getting lost in some cases (i.e., people didn't know they were there), so what you did on these works great for now.